README.md

```diff
@@ -32,6 +32,20 @@ To get a local copy up and running:
 
```

Comment:
- [ ] On your readme file, the formatting seems to be a little off:

![image](https://user-images.githubusercontent.com/57421823/170247402-d58f6ef2-f9b2-4411-9cd6-48bda13a4677.png)

Kindly correct it, so the project is well documented.